sdk/src/main/java/io/opentdf/platform/sdk/CryptoUtils.java

```diff
@@ -39,6 +40,30 @@ public static KeyPair generateRSAKeypair() {
         return kpg.generateKeyPair();
     }
 
+    public static KeyPair generateECKeypair(String curveName) {
+        KeyPairGenerator kpg;
+        try {
+            kpg = KeyPairGenerator.getInstance("EC");
+            ECGenParameterSpec ecSpec = new ECGenParameterSpec(curveName);
+            kpg.initialize(ecSpec);
+        } catch (NoSuchAlgorithmException | InvalidAlgorithmParameterException e) {
+            throw new SDKException("error creating EC keypair", e);
+        }
+        return kpg.generateKeyPair();
+    }
+
+    public static String getPublicKeyPEM(PublicKey publicKey) {
+        return "-----BEGIN PUBLIC KEY-----\r\n" +
+                Base64.getMimeEncoder().encodeToString(publicKey.getEncoded()) +
+                "\r\n-----END PUBLIC KEY-----";
+    }
+
+    public  static String getPrivateKeyPEM(PrivateKey privateKey) {
+        return "-----BEGIN PRIVATE KEY-----\r\n" +
+                Base64.getMimeEncoder().encodeToString(privateKey.getEncoded()) +
+                "\r\n-----END PRIVATE KEY-----";
+    }
```

Comment:
I think there is a PEM package we should use here

Reply:
I guess this works fine.  There is also org.bouncycastle.openssl.jcajce.JcaPEMWriter